packages/server/src/api/routes/public/index.ts

```diff
@@ -29,7 +29,7 @@ function getApiLimitPerSecond(): number {
   return parseInt(env.API_REQ_LIMIT_PER_SEC)
 }
 
-if (!env.isTest()) {
+/*if (!env.isTest()) {
```

Comment:
Noticed this commented out, this will need added back in, not sure if you're still experiencing issues with this, might be worth trying pulling the latest stuff and seeing if its still not connecting.

Reply:
I am still experiencing issues with this part of the code, but I shouldn't have committed it. I reverted the change.